### PR TITLE
Duplicate tv and moves in person details-view

### PR DESCRIPTION
**Pull request by AndreasPB**
### Before reporting

- [X] I have tested with the lastest _master_

### Describe the bug

image

### To Reproduce

1. Go to person/10980
2. Scroll to Series
3. Look at duplicates

### Expected behavior

image

### Additional context

_No response_

### CODE REVIEW

File: frontend/src/routes/person/[id].svelte [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <script lang="ts">
-    import { removeContentWithMissingImagePath, routeToPage, sortListByPopularity  } from '../../utils'
+    import { removeContentWithMissingImagePath, routeToPage, sortListByPopularity, removeDuplicates, removeDuplicates } from '../../utils'
```
Comment (Primdahl26) [inline frontend/src/routes/person/[id].svelte:2]: Why is `removeDuplicates` imported twice?

**Review by Primdahl26 — CHANGES_REQUESTED** [review]
A LGTM, just needs the duplicate import removed

File: frontend/src/routes/person/[id].svelte (hunk 1 above)
Comment (AndreasPB) [reply frontend/src/routes/person/[id].svelte:2]: Uhm, yes.

Comment (AndreasPB) [reply frontend/src/routes/person/[id].svelte]: Removed.

**Review by Primdahl26 — APPROVED** [review]

**Pull request merged**